test/suites/ekuiper/main_test.go

```diff
@@ -81,23 +57,8 @@ func TestMain(m *testing.M) {
 		ascSnap,
 	)
 
-	testSecretsInterface = false
-	exitCode = m.Run()
-
-TEARDOWN:
-	log.Println("[TEARDOWN]")
-
-	utils.SnapDumpLogs(nil, start, ekuiperSnap)
-	utils.SnapDumpLogs(nil, start, "edgexfoundry")
-
-	utils.SnapRemove(nil,
-		ekuiperSnap,
-		"edgexfoundry",
-		deviceVirtualSnap,
-		ascSnap,
-	)
-
-	os.Exit(exitCode)
+	code = m.Run()
+	os.Exit(code)
```

Comment:
@MonicaisHer Teardown is not called here, so the snaps aren't removed and logs aren't collected.

Reply:
`teardown()` is missed here

Reply:
this will be solved by https://github.com/canonical/edgex-snap-testing/pull/115